### PR TITLE
feat: add anti-swear system with `leo-profanity`

### LINKED ISSUE #316: [FEAT] - Add an anti-swear system
### What feature would you like to see?

Add an anti-swear system. We can use the [`leo-profanity`](https://github.com/jojoee/leo-profanity) library for that. It works very well with a lot of words and it doesn't get confused with stuff like `assignment`, which *could* get blocked for the first three letters.

### Additional information

I don't think so.

### PR DESCRIPTION
**Pull request by Panquesito7**
# Pull Request

## Description

- Add anti-swear system with `leo-profanity`.

## Related Issue

Closes #316

## Scope

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [x] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [ ] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation

For _any_ version updates, please verify if the documentation page needs an update. If it does, please create an issue to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

**Comment by j-dogcoder** [comment]
Hey @Panquesito7, This should be a Minor Version update.

File: main.js [hunk 1]
```diff
@@ -195,6 +196,21 @@ client.on('ready', async () => {
...
+    if (filter.check(message.content)) {
+      message.delete()
+      const warning = await message.channel.send(
+        `${message.author}, do not use bad words in chat!`
+      )
+      setTimeout(warning.delete, 4500)
+    }
+  }
+})
+
```
Comment (j-dogcoder) [inline main.js:213]: Hey! This should live in a file inside of the features directory.

**Review by j-dogcoder — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (Panquesito7) [reply main.js:213]: > Hey! This should live in a file inside of the features directory.

Alright! I'll work on that, thanks. 🙂

**Review by j-dogcoder — APPROVED** [review]

**Comment by Panquesito7** [comment]
Could the `hacktoberfest-accepted` label be added, please? Thanks. 🙂

**Comment by j-dogcoder** [comment]
Sorry!

**Comment by Panquesito7** [comment]
> Sorry!

It's alright, thanks! 😄

**Pull request merged**